### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by berkkrkc09** [unchihugo/FluentFlyout#30]
**Purpose:**  
My notifications are displayed on my second monitor because I don't want any distractions on my main monitor. I was going to do the same with flyouts but it is not an option yet. This feature would be a great solution for intrusion just like the already built-in compact layout.

**Description:**  
I would love to be able to set the screen that the flyout will display on with individual toggles. Example: I would set media and next up flyout to display on 2nd monitor but the lock keys flyout would be on my 1st monitor.

**Additional context:**  
I use a Windhawk mod to move the notifications to my second monitor. It would be a great combo with this app if it had monitor selection.

**Comment by agbrettpittman** [comment]
I found Fluent Flyout today and absolutely love it! Being able to choose a monitor would be super nice :)

**Comment by WobblySausage** [comment]
Yeah I'd pay for the app if it had multi monitor selection, doesn't work for me in at the moment because gaming and full screen apps make it redundant unfortunately. 

To be able to add this to another screen would be perfect

**Comment by WobblySausage** [comment]
Updated but can't see this new feature as shown in the mp4 files uploaded. I can see flyout position, widget position. Using 2.4.1

**Comment by unchihugo** [comment]
> Updated but can't see this new feature as shown in the mp4 files uploaded. I can see flyout position, widget position. Using 2.4.1

It's not released as an update yet. If you don't want to wait, you can test out the latest updates by building it yourself!

**Comment by unchihugo** [comment]
Just released v2.5.0, which gives you the ability to change which monitor the flyouts (and widget) displays on. Currently, the new setting applies to all three flyouts we have. We'll look into being able to manually choose a display for each flyout in the future.

**Comment by WobblySausage** [comment]
Excellent. Purchased!
